### DISCUSSION TITLE
Why I can't see creating branch button?

### DISCUSSION THREAD

**Discussion by HeavySnowJakarta** [community/community#36762] [New to GitHub] [answered]
I want to create a branch, but where is the creating button?
![IMG_3742](https://user-images.githubusercontent.com/54460050/196907815-d6a39069-02ce-4cdd-9450-6a5765f74d0e.jpeg)

**Comment by DSDmark** [comment]
Hey @HeavySnowJakarta,

in my case, I can see the button for **create new branch**:-

- ![Screenshot from 2022-10-20 21-28-46](https://user-images.githubusercontent.com/85744816/197000479-ff907174-eb4a-4928-8546-8c901b9f22f6.png)

I think it is not your repo. You're seeing another person repo.

**Reply by HeavySnowJakarta** [reply]
> Hey @HeavySnowJakarta,
> 
> 
> 
> in my case, I can see the button for **create new branch**:-
> 
> 
> 
> - ![Screenshot from 2022-10-20 21-28-46](https://user-images.githubusercontent.com/85744816/197000479-ff907174-eb4a-4928-8546-8c901b9f22f6.png)
> 
> 
> 
> I think it is not your repo. You're seeing another person repo.

So I can't create branches for others’ repo? How can I contribute to it then?

**Reply by DSDmark** [reply]
@HeavySnowJakarta are you kidding me I was thinking a lot why is your **new branch** button is not show and you seeing other person repo. Oo god just kill me.... 😂

**Comment by DSDmark** [comment, marked as answer]
Dude... you need to **fork** that branch first
for example, if you want to quick start how to do fork, then you can do like this:-

### it's your branch ok. I want to contribute in your repo for example:-

### For contribution, click on **Fork button**

- ![Screenshot from 2022-10-21 07-53-34](https://user-images.githubusercontent.com/85744816/197096734-e004adea-3d14-47b7-b5b4-86598fd69ccc.png)

----
now after click on that **Fork button,** this creates a new repo in your account like this:- 

- ![Untitled (1)](https://user-images.githubusercontent.com/85744816/197096903-5447bdbf-f256-46e1-8a8f-6b7e24ed56ff.png)

----
now I'm ready to contribute in your repo.

- ![Untitled](https://user-images.githubusercontent.com/85744816/197097429-eb201378-1f1e-42c3-bc60-74867e42f3d9.png)

Once you're done with that, then the second step is to create a **pull request,** I not telling you how to do that. If you want to tell me I write all steps how to do that, ok.

-----

#### also it's good thing to read**official documents first**.
Here are **official** docs :- 
[**Pull Request**](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
[**Fork**](https://docs.github.com/en/get-started/quickstart/fork-a-repo)

**Comment by ossanna16** [comment]
Thank you for asking a good question @HeavySnowJakarta, I'm glad you found our community 🙂

**Reply by HeavySnowJakarta** [reply]
> Thank you for asking a good question @HeavySnowJakarta, I'm glad you found our community 🙂 

Feeling surprised that you are friendly to such a  stupid question. I will have a try later and understand the workflow better.  (☆^O^☆)

**Reply by DSDmark** [reply]
Hey @HeavySnowJakarta,
I hope you get your answer. If you get your answer, then you should close that question by click **accept answer** button, or you can write the answer and accept own answer.
